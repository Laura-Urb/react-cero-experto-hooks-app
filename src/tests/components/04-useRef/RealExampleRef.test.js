import { shallow } from 'enzyme';
import { RealExampleRef } from '../../../components/04-useRef/RealExampleRef';
import React from 'react';

describe('Pruebas RealExampleRef', () => {
  const wrapper = shallow(<RealExampleRef />);

  test('debe de mostrarse correctamente ', () => {
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find('MultipleCustomHooks').exists()).toBe(false);
  });
  test('debe de mostrarse correctamente MultipleCustomHooks', () => {
    wrapper.find('button').simulate('click');
    expect(wrapper.find('MultipleCustomHooks').exists()).toBe(true);
  });
});
