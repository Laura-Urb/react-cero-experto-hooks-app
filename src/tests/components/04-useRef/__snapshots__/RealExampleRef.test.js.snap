// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas RealExampleRef debe de mostrarse correctamente  1`] = `
<div>
  <h1>
    Real Example Ref
  </h1>
  <hr />
  <button
    className="btn btn-primary mt-5"
    onClick={[Function]}
  >
    Show/hide
  </button>
</div>
`;
